README.md

```diff
@@ -780,11 +780,13 @@ OPTIONS
 
   -v, --verbose                        Verbose output
 
-  -w, --workspace=workspace            Specify the Adobe Developer Console Workspace name to import the configuration
-                                       from
+  -w, --workspace=workspace            Specify the Adobe Developer Console Workspace name or Workspace id to import the
```

Comment:
We should not be manually updating the readme, this is a generated file at the time of an npm publish ... otherwise we can expect merge issues to happen.